octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,13 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        System.out.println("Sum of firstNumber and secondNumber is: " + add(10, 20));
+    }
+
```

Comment:
[Severity: Minor] The method currently only performs addition. Consider implementing other arithmetic operations (subtraction, multiplication, division) as separate methods.
This will fulfill the task of creating a basic calculator that can perform multiple operations.
Rationale: A complete calculator should handle various operations, enhancing its functionality.
Reference: https://www.geeksforgeeks.org/methods-in-java/